### DISCUSSION TITLE
How use a google font inside my element?

### DISCUSSION THREAD

**Discussion by andersoncampolina** [lit/lit#4036] [Q&A]
I wanna use google fonts inside my Lit element, but to do it, I want to import it from the index.html level. I wanna use it directly inside the component, so the user of my web-component dont need to handle this import himself.
Is that possible?

**Comment by VandeurenGlenn** [comment]
@andersoncampolina 
You just put the link inside your components template or you handle adding it to body once your component is loaded.

https://github.com/VandeurenGlenn/custom-elements/blob/78efffeca91ef023c2ad8fa43cfe23109b424c1d/src/theme/theme.ts#L55
